kuma/javascript/src/header/search.jsx

```diff
@@ -9,16 +9,16 @@ type Props = {
     initialQuery: string,
 };
 
-export default function Search(props: Props) {
+export default function Search({ initialQuery }: Props) {
     const locale = getLocale();
 
     // After our first render, set the input field's initial value
     const inputfield = useRef(null);
     useEffect(() => {
-        if (inputfield.current && props.initialQuery) {
-            inputfield.current.value = props.initialQuery;
+        if (inputfield.current && initialQuery) {
+            inputfield.current.value = initialQuery;
         }
-    }, [props.initialQuery]);
+    }, [initialQuery]);
```

Comment:
Another eslint error fix-- `props` will change when any props change (in this case there's only one prop right now so it doesn't really matter) so the fix is to destructure props and refer to the specific props inside `useEffect`.